src/coreclr/jit/emitarm64.cpp

```diff
@@ -5579,6 +5593,21 @@ void emitter::emitIns(instruction ins)
     appendToCurIG(id);
 }
 
+/*****************************************************************************
+ *
+ *  Add a SETFFR instruction: initialize first-fault register to all true.
+ */
+
+void emitter::emitInsSve_SetFFR()
```

Comment:
We should probably reuse `emitIns_I()` for it. It encodes the instructions that don't take registers.